config/config_file.go

```diff
@@ -45,7 +45,13 @@ func LoadFile(path string) (*File, error) {
 		path = fmt.Sprintf("%s%s", homedir, path[1:])
 	}
 
-	iniFile, err := ini.Load(path)
+	opts := ini.LoadOptions{
```

Comment:
nit: `opts` is not used anywhere after line 54, can you inline it?

Reply:
Done.